apps/entry/views.py

```diff
@@ -224,3 +232,32 @@ def get_serializer_context(self):
         context = super().get_serializer_context()
         context['queryset'] = self.get_queryset()
         return context
+
+
+class EntryCommentViewSet(viewsets.ModelViewSet):
+    serializer_class = EntryCommentSerializer
+    permission_classes = [permissions.IsAuthenticated,
+                          ModifyPermission]
+    filter_backends = (django_filters.rest_framework.DjangoFilterBackend,)
+    filterset_class = EntryCommentFilterSet
+
+    def get_queryset(self):
+        return EntryComment.get_for(self.request.user)
+
+    @action(
+        detail=True,
+        url_path='resolved',
```

Comment:
Better name this `resolve`.

Reply:
Done

@AdityaKhatri Please change the API URL in client to
```js
/api/v1/entry-comments/<entry-comment-id>/resolve/
```